### DISCUSSION TITLE
考虑到自定义主题需要较大的区域展示

### DISCUSSION THREAD

**Discussion by pdog18** [pdog18/Soak#7] [Announcements]
所以考虑将自定义主题界面拆分到单独的界面（甚至是 repo），这样剩下的其他设置就可以变得非常紧凑了。

**Comment by pdog18** [comment]
拆分后，soak 的自定义主题将用主题库来替代，但是主题库考虑可以从 issue 或 Discussions 中直接更新，而不需要维护代码。

**Comment by yanhuacuo** [comment]
有没有计划写一写 wiki ？
特别是关于「主题」这块儿的参数功能。
我在做一个 weasel-tool ，但是发现一堆参数要自己试功能。
大佬的 weasel 添加新版参数了吗？

目前抽出了 37 个新参数：https://wubi98.gitee.io/2023/06/16/2023-06-16-01.rimetool/

今天刚写完【进程守护】 ，其它界面还没搭起来。

![PIC](https://raw.githubusercontent.com/yanhuacuo/b-sty/master/1.png)

![PIC](https://raw.githubusercontent.com/yanhuacuo/b-sty/master/2.png)

![PIC](https://raw.githubusercontent.com/yanhuacuo/b-sty/master/3.png)

![PIC](https://raw.githubusercontent.com/yanhuacuo/b-sty/master/4.png)

**Reply by pdog18** [reply]
主题的参数吗？您指的是这个么？ 

https://github.com/fxliang/weasel/releases/tag/0.14.3_new_features

**Reply by pdog18** [reply]
可能我不太理解您提到的 wiki 是什么，可能还需要您耐心解释一下。
我大概理解成每个参数的作用，但是我记得目前有似乎这个方面的信息，只是可能要找一找。
如果我理解错了还需要您耐心解释一下。

**Reply by yanhuacuo** [reply]
刚看到，新 wiki 页已经有了：

https://github.com/rime/weasel/wiki/Weasel-%E5%AE%9A%E8%A3%BD%E5%8C%96

**Reply by yanhuacuo** [reply]
看到 「Electron App」 的 TODO 项了，期待啊，

**Reply by pdog18** [reply]
这个真不错

**Reply by pdog18** [reply]
一直都有这个，只是优先级特别低，低到都不确定会不会做
